src/ast/nodes/ExportDefaultDeclaration.js

```diff
@@ -35,20 +35,22 @@ export default class ExportDefaultDeclaration extends Node {
 
 		// paren workaround: find first non-whitespace character position after `export default`
 		let declaration_start;
+		let id_index;
```

Comment:
"index" is a very vague name that is easily misinterpreted. How about `id_insertPos` or something similar that better describes what this variable is used for?